extensions/sigstore.md

```diff
@@ -0,0 +1,51 @@
+# Signature Extension: Sigstore
+
+July 20, 2023
+
+Version: TBD
+
+Status: WIP
+
+This document describes the
+[signature extension](../envelope.md#signature-extensions) schema for the
+Sigstore ecosystem.
+
+## Definition
+
+The Sigstore extension's type must be set to `https://sigstore.dev`.
```

Comment:
nit: Should this be versioned? (IDK)

Reply:
Yeah, I think it probably should be. I wasn't sure how much to specify here, I wanted to hear from others.

Reply:
I think using the media type directly (with version) might work best rather than using a URI here and a media type elsewhere.